### PR TITLE
Add libdrm-tests to dbg package

### PR DESCRIPTION
**Pull request by m-iwanicki**
libdrm-tests contains e.g. 'modetest' which is useful for drm debugging

### CODE REVIEW

File: meta-zarhus-distro/recipes-zarhus/packagegroups/packagegroup-zarhus.bb [hunk 1]
```diff
@@ -23,4 +23,6 @@ RDEPENDS:${PN}-dbg = " \
     libgpiod \
     libgpiod-tools \
     devmem2 \
+    libdrm \
```
Comment (DaniilKl) [inline meta-zarhus-distro/recipes-zarhus/packagegroups/packagegroup-zarhus.bb]: @m-iwanicki , should it be in `-dbg` package? This is described as `Userspace interface to the kernel DRM services`, so you can add it to `-graphics` package for example.

Comment (macpijan) [reply meta-zarhus-distro/recipes-zarhus/packagegroups/packagegroup-zarhus.bb]: I wonder if we need to explicitly install it here in the first place. Isn't this already pulled in by the weston, etc?

Comment (m-iwanicki) [reply meta-zarhus-distro/recipes-zarhus/packagegroups/packagegroup-zarhus.bb]: In manifest there is `libdrm2` package.
I wanted libdrm mostly for `modeset` and other testing tools. Manifest with this change contains:

```
libdrm-amdgpu1 cortexa55 2.4.120
libdrm-etnaviv1 cortexa55 2.4.120
libdrm-tests cortexa55 2.4.120
libdrm2 cortexa55 2.4.120
```

Comment (m-iwanicki) [reply meta-zarhus-distro/recipes-zarhus/packagegroups/packagegroup-zarhus.bb]: Removed `libdrm`, works with only `libdrm-tests`

**Review by macpijan — APPROVED** [review]

**Pull request merged**